src/pages/index.js

```diff
@@ -17,12 +21,13 @@ const profileDescriptionInput = document.querySelector(
 );
 const profileAddButton = document.querySelector(".profile__add-button");
 const profileForm = document.forms["profile-form"];
+const avatarForm = document.forms["change-profile-image-form"];
+const editProfileImageIcon = document.querySelector(".profile__image-pencil");
 /*                                                                          */
 /*                                ADD CARD CONSTANTS                        */
 /*                                                                          */
 const addCardForm = document.forms["add-card-form"];
-
-
+const changeAvatarForm = document.forms["change-profile-image-form"];
```

Comment:
`const changeAvatarForm = document.forms["change-profile-image-form"];
`

You've already defined the avatar form in line 24. Please, use it